manifests/a/AceStream/AceStream/3.1.74/AceStream.AceStream.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AceStream.AceStream
+PackageVersion: 3.1.74
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://download.acestream.media/products/acestream-full/win/latest
+  InstallerSha256: B506A7917FE91E085773194FF5D291EC59459DFB4AFC0B2970A5F75803DC953F

```

```suggestion
  InstallerSha256: B506A7917FE91E085773194FF5D291EC59459DFB4AFC0B2970A5F75803DC953F
  ProductCode: AceStream
```
Not mandatory, but better to have one for better version matching in upgrade scenario